modules/frontend/config.go

```diff
@@ -79,6 +79,7 @@ func (cfg *Config) RegisterFlagsAndApplyDefaults(string, *flag.FlagSet) {
 			ConcurrentRequests:    defaultConcurrentRequests,
 			TargetBytesPerRequest: defaultTargetBytesPerRequest,
 			IngesterShards:        3,
+			MaxSpansPerSpanSet:    1_000,
```

Comment:
Arbitrary number; to decide

Reply:
100 feels like a good default to me? that's still quite a few spans.

By putting it in this struct it is already configurable! Please add some docs here:

https://github.com/grafana/tempo/blob/main/docs/sources/tempo/configuration/_index.md?plain=1#L642-L643

Reply:
> I suppose the manifest YAML is automatically generated?

Nope. If you want to add that to CI, please do.